api/src/pcapi/routes/pro/venues.py

```diff
@@ -39,6 +40,14 @@
 @spectree_serialize(response_model=GetVenueResponseModel, api=blueprint.pro_private_schema)
 def get_venue(venue_id: str) -> GetVenueResponseModel:
     venue = load_or_404(Venue, venue_id)
+
+    if venue.bannerUrl is not None:
+        if venue.bannerMeta is None:
+            venue.bannerMeta = {}
+
+        if "crop_params" not in venue.bannerMeta:
+            venue.bannerMeta["crop_params"] = DO_NOT_CROP
+
     check_user_has_access_to_offerer(current_user, venue.managingOffererId)  # type: ignore [attr-defined]
```

Comment:
```suggestion

    check_user_has_access_to_offerer(current_user, venue.managingOffererId)  # type: ignore [attr-defined]

    if venue.bannerUrl is not None:  # type: ignore [attr-defined]
        if venue.bannerMeta is None:  # type: ignore [attr-defined]
            venue.bannerMeta = {}  # type: ignore [attr-defined]

        if "crop_params" not in venue.bannerMeta:  # type: ignore [attr-defined]
            venue.bannerMeta["crop_params"] = DO_NOT_CROP  # type: ignore [attr-defined]
```

mais je comprends pas pourquoi

Reply:
Malheureusement ça ne suffit pas on va attendre le retour de Jérémie sur la PR

Reply:
Du coup c'était ok en rajoutant `# type: ignore [attr-defined]` sur les lignes `bannerMeta` et `bannerUrl`